catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -228,6 +235,8 @@
 
     private static final XMLUtils XML_UTILS = XMLUtils.getInstance();
 
+    private static final int DEFAULT_PAGE_SIZE = 20;
```

Comment:
I matched the default used by the paging Iterables. I do not know if there was extensive (or any) empirical study to support that page size.